src/main/java/org/ihiw/management/repository/UploadRepository.java

```diff
@@ -31,7 +31,9 @@
     List<Upload> findAll();
 
     List<Upload> findAllById(List<Long> ids);
-
+    
+    @Query("select upload from Upload upload where upload.parentUpload.id = ?1")
```

Comment:
With Spring JPA it shouldn't even be necessary to specify the SQL code... But let's leave it for now.